SU2_CFD/src/solvers/CIncEulerSolver.cpp

```diff
@@ -1597,7 +1597,7 @@ void CIncEulerSolver::Source_Residual(CGeometry *geometry, CSolver **solver_cont
 
       /*--- Get the physical time. ---*/
       su2double time = 0.0;
-      if (config->GetTime_Marching()) time = config->GetPhysicalTime();
+      if (config->GetTime_Marching() != TIME_MARCHING::STEADY) time = config->GetPhysicalTime();
```

Comment:
Watch out for: implicitly relying that STEADY evaluates to zero/false and everything else to >0 and therefore true.